Samples/Dinner/docker/collector-config.yaml

```
@@ -0,0 +1,24 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: otel-collector:4317
+
+exporters:
+  logging:
+    logLevel: debug
```

Just a heads-up that this changed to `verbosity: detailed` in a recent `otel-collector` release. `logLevel: debug` is not supported anymore. Since we're using the `latest` tag in `docker-compose.yml` we should update it accordingly.

```suggestion
    verbosity: detailed
```

---

Nice, thanks! my docker setup os broken on my main box so I've not verified those yet :(